interview-questions.md

```diff
@@ -8,63 +8,112 @@ Then, research the question to expand on your answer. Even if you feel you have
 
   Researched answer:
-
+    In TDD, tests are written before the actual code that is being tested. You want to write the code as minimally as you can, and once it passes, you want to refactor it to make it as clean and concise as possible. If you don't use TDD, you are unable to discover your errors in code before the full functionality has been develped. TDD allows you to code incrementally so you don't miss anything before the code is fully finished.
 
 
 7. What is something we did in class this week you found helpful?  
 
   Your answer:
+    I like how active other instructors are in the chat box during lecture. It's helpful to see additional knowledge and perspective. It's also makes it easy to ask simple questions in the chat box. The chat box is a great tool that allows a greater form of communication without stopping the flow of the class.
+
+    I like how instructors randomly drop into our breakout rooms. I'm not shy to ask for help, but it feels nice to know that the instructors care about our learning. Sometimes they help with things I didn't know I needed help with, like when Andee taught us how to string multiple methods onto each other to save memory.
+
+    I like the pseudocode that is written before the actual code, so we can understand the logic behind the problem before we assign code to it.
+
+    I like knowing who I'm going to be paired up with the next day. I've enjoyed working with everybody so far, but it's helpful for me to see who I'm working with so I can anticipate what my breakout time will be like. 
+
+    I like that Sarah gives us our return time from breaks. Sometimes I forget what time to return haha.
+
+    I love that the instructors pair program during lecture. I think it shows a great example for us. It also seems to allow the navigator instructor to give their undivided attention to the lecture notes and class. 
+
+  Some things I would like more of:
+    I wish that before moving onto the next practice problem in lecture, we would be given a few seconds to soak in the content. Often times, I'm still rushing to complete the notes by the time we move onto the next problem, and I don't have a chance to look at the code as a whole.
+
+    I like being able to see the most recent lines of the terminal while coding, so it's helpful to me when the terminal is placed on the lower corner of the screen while we're on Atom. That way, even if we're focused on the Atom window, I can still see the last lines of the terminal in the bottom right hand corner.
+
+    It would be nice to be pulled out of the breakroom or notified when lunch comes around. I've heard from several people that we sometimes cut into our lunch time because we don't realize what time it is.
+
+    Sarah is a great instructor, but I think she can go through some slides a little too quickly :P
```

Comment:
Love this feedback!!! Its so helpful to us when our students can provide this level of feedback and I encourage you to always reach out to us and provide any more you may have as you go through the course :)